sycl/test-e2e/ESIMD/lsc/lsc_usm_2d.cpp

```diff
@@ -61,7 +61,7 @@ int main() {
                            cache_hint::uncached>(
                 input, (data_width * sizeof(int)) - 1, data_height - 1,
                 (data_pitch * sizeof(int)) - 1, x, y);
-            auto data = lsc_load2d<int, Width, Height, NumBlocks, false, false,
+            auto data = lsc_load_2d<int, Width, Height, NumBlocks, false, false,
```

Comment:
Shouldn't the call at L60 also be changed? lsc_prefetch2d

Reply:
Yes.